ui/sdk/src/hamilton_sdk/adapters.py

```diff
@@ -250,6 +250,7 @@ def post_node_execute(
         result: Optional[Any],
         task_id: Optional[str] = None,
     ):
+        print("I got to hamilton tracker")
```

Comment:
We never make it to here, so the UI shows the nodes as still running no matter what happened.